spec/directions/bake_chapter_review_exercises/v1_spec.rb

```diff
@@ -47,34 +50,8 @@
     )
   end
 
-  let(:metadata) do
-    <<~HTML
-      <div data-type="metadata" style="display: none;">
-        <h1 data-type="document-title" itemprop="name">A Title</h1>
-        <div class="authors">
-          <span id="author-1#" ><a>OpenStaxCollege</a></span>
-        </div>
-        <div class="publishers">
-          <span id="publisher-1#"><a>OpenStaxCollege</a></span>
-        </div>
-        <div class="print-style">
-          <span data-type="print-style">ccap-book</span>
-        </div>
-        <div class="permissions">
-          <p class="copyright">
-            <span id="copyright-holder-1#"><a>OSCRiceUniversity</a></span>
-          </p>
-          <p class="license">
-            <a>CC BY</a>
-          </p>
-        </div>
-        <div itemprop="about" data-type="subject">Book Title</div>
-      </div>
-    HTML
-  end
-
   it 'works' do
-    described_class.new.bake(chapter: book_with_review_exercises.chapters.first, metadata_source: metadata, append_to: append_to)
+    described_class.new.bake(chapter: book_with_review_exercises.chapters.first, metadata_source: metadata_element, append_to: append_to, klass: 'review-exercises')
```

Comment:
`metadata_element` is a factory helpers method that returns some generic metadata